modules/nlu/src/backend/api.ts

```diff
@@ -209,11 +209,23 @@ export default async (bp: typeof sdk, state: NLUState) => {
     const { botId, id } = req.params
     const botEngine = state.nluByBot[botId].engine1 as ScopedEngine
     await botEngine.storage.deleteEntity(id)
-    scheduleSyncNLU(req.params.botId)
+    scheduleSyncNLU(botId)
 
     res.sendStatus(204)
   })
 
+  router.post('/train', async (req, res) => {
+    try {
+      const { botId } = req.params
+      const botEngine = state.nluByBot[botId].engine1 as ScopedEngine
+      scheduleSyncNLU(req.params.botId)
```

Comment:
those 2 lines are doing the same thing. 

I imanine you wanted to use state.nluBybot[botI].engine as Engine2